pkgs/applications/networking/feedreaders/yarr/default.nix

```diff
@@ -2,16 +2,16 @@
 
 buildGoModule rec {
   pname = "yarr";
-  version = "2.3";
+  version = "2.4";
 
   src = fetchFromGitHub {
     owner = "nkanaev";
     repo = "yarr";
     rev = "v${version}";
-    hash = "sha256-LW0crWdxS6zcY5rxR0F2FLDYy9Ph2ZKyB/5VFVss+tA=";
+    hash = "sha256-ZMQ+IX8dZuxyxQhD/eWAe4bGGCVcaCeVgF+Wqs79G+k=";
   };
 
-  vendorHash = "sha256-yXnoibqa0+lHhX3I687thGgasaVeNiHpGFmtEnH7oWY=";
+  vendorHash = null;
```

Comment:
I don't know what this means, so I guess I will not merge this now. Can someone with go packaging experience decide?

Reply:
That's OK, since upstream vendored dependencies, see https://github.com/nkanaev/yarr/commit/5e46f1480e6e8eb2e776ea85d64adef13cd8fb4f

Reply:
Okay, then I'll merge this!